src/Console/ConsoleOptionParser.php

```diff
@@ -589,6 +592,39 @@ public function addSubcommand($name, array $options = [])
     }
 
     /**
+     * @param array $options Options
+     * @return array
+     */
+    protected function _mergeSubcommandHelpToParserDescription($options)
+    {
+        if (!$options['help']) {
+            return $options;
+        }
+
+        if ($options['parser'] && is_object($options['parser'])) {
```

Comment:
ConsoleOptionParser doesnt have an interface, other code checks for the concrete class.
Maybe we could add an interface for a future release here? That would be great.